examples/angular/src/app/alertWrapper/alert.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, CUSTOM_ELEMENTS_SCHEMA, input } from '@angular/core';
+
+@Component({
+  selector: 'app-alert',
+  standalone: true,
+  schemas: [CUSTOM_ELEMENTS_SCHEMA],
+  templateUrl: './alert.component.html',
+  styleUrls: ['./alert.component.css'],
```

Comment:
Do we need any of these css files?

Reply:
Strictly speaking, no. I have them in there for completeness, but it's also possibly confusing since I'm not doing any styling at all. Should I do some styling? Should we find out how wacky we can get and how the shadow DOM impacts these styles if they are present?

Reply:
Totally valid!